float64 s
float64 s_d
float64 s_dd
float64 s_ddd
float64 l
float64 l_d
float64 l_dd
float64 l_ddd
float64 l_ds
float64 l_dds
float64 l_ddds
float64 ds

